src/helpers/query/orgs.js

```diff
@@ -459,8 +385,59 @@ export const orgFetchAll = async (
       return [];
     }
   }
+
+  // Add creator data if requested
+  if (includeCreators) {
+    try {
+      // Extract unique creator IDs
+      const creatorIds = [...new Set(
+        orgs
+          .map(org => org.createdBy)
+          .filter(Boolean)
+      )];
+
+      if (creatorIds.length > 0) {
+        // Fetch creator data in batch
+        let creatorsData = [];
+        try {
+          creatorsData = await fetchDocumentsById(FIRESTORE_COLLECTIONS.USERS, creatorIds, ['displayName']);
```

Comment:
Let's also fetch the name property so that we can use that as a fallback if there's no display name instead of showing the ID of the user. So if there's no display name or name property, then we just show the two dashes (--). 

<img width="554" height="145" alt="Image" src="https://github.com/user-attachments/assets/5b55dffd-20f5-4411-ba13-888303742026" />